src/router.js

```diff
@@ -0,0 +1,36 @@
+import Vue from 'vue'
+import Router from 'vue-router'
+import Home from '@/views/Home.vue'
+import Callback from '@/views/Callback.vue'
+
+Vue.use(Router)
+
+const router = new Router({
+  mode: 'history',
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      component: Home
+    },
+    {
+      path: '/callback',
+      name: 'callback',
+      component: Callback
+    }
+  ]
+})
+
+// very basic global guard
+router.beforeEach((to, from, next) => {
+  if (to.name == 'callback') { // check if "to"-route is "callback" and allow access
```

Comment:
I believe this is necessary so Auth0 can always reach, even if user is supposed to be blocked.

Reply:
Ohhh. Gotcha.